go/scheduler/api/api.go

```diff
@@ -67,25 +67,30 @@ type CommitteeNode struct {
 type CommitteeKind uint8
 
 const (
+	// KindCompute is a compute committee.
+	KindInvalid CommitteeKind = 0
+
 	// KindExecutor is an executor committee.
-	KindExecutor CommitteeKind = 0
+	KindExecutor CommitteeKind = 1
 
 	// KindStorage is a storage committee.
-	KindStorage CommitteeKind = 1
+	KindStorage CommitteeKind = 2
 
 	// KindTransactionScheduler is a transaction scheduler committee.
-	KindTransactionScheduler CommitteeKind = 2
+	KindTransactionScheduler CommitteeKind = 3
 
 	// KindMerge is a merge committee.
-	KindMerge CommitteeKind = 3
+	KindMerge CommitteeKind = 4
 
 	// MaxCommitteeKind is a dummy value used for iterating all committee kinds.
-	MaxCommitteeKind = 4
+	MaxCommitteeKind = 5
 )
 
 // NeedsLeader returns if committee kind needs leader role.
 func (k CommitteeKind) NeedsLeader() bool {
 	switch k {
+	case KindInvalid:
+		panic(fmt.Errorf("scheduler: KindInvalid should not be queried for a leader"))
```

Comment:
Panicking here seems dangerous, what if an invalid kind is received over the wire?